### PR TITLE
Various updates

### PR DESCRIPTION
**Pull request by curiosityseeker**

### CODE REVIEW

File: apparmor.d/groups/kde/kwin_wayland [hunk 1]
```diff
@@ -68,10 +71,19 @@ profile kwin_wayland @{exec_path} flags=(attach_disconnected mediate_deleted) {
   owner /var/lib/sddm/.config/kwinrc.lock rwk,
   owner /var/lib/sddm/.config/kwinrc{,.@{rand6}} rwl -> /var/lib/sddm/.config/#@{int},
 
-  owner @{user_cache_dirs}/{,plasma-svgelements} r,
+  owner @{user_cache_dirs}/#@{int} rw,
+  owner @{user_cache_dirs}/{,plasma-svgelements} r
+  owner @{user_cache_dirs}/fontconfig/*-le64.cache-@{int} w,
+  owner @{user_cache_dirs}/fontconfig/*-le64.cache-@{int}{,TMP-@{rand6},NEW,LCK} w,
```
Comment (roddhjav) [inline apparmor.d/groups/kde/kwin_wayland:77]: Use `abstractions/fontconfig-cache-write` instead

File: apparmor.d/groups/kde/kwin_wayland [hunk 2]
```diff
@@ -68,10 +71,19 @@ profile kwin_wayland @{exec_path} flags=(attach_disconnected mediate_deleted) {
...
   owner @{user_cache_dirs}/icon-cache.kcache rw,
   owner @{user_share_dirs}/kscreen/* r,
   owner @{user_cache_dirs}/ksycoca5_* r,
+  owner @{user_cache_dirs}/kwin/qmlcache/#@{int} rw,
+  owner @{user_cache_dirs}/kwin/qmlcache/*.qmlc{,.@{rand6}} rw,
+  owner @{user_cache_dirs}/kwin/qmlcache/*.qmlc.@{rand6} l -> @{user_cache_dirs}/kwin/qmlcache/#@{int},
+  owner @{user_cache_dirs}/plasma-svgelements.lock rwk,
+  owner @{user_cache_dirs}/plasma-svgelements.@{rand6} rw,
+  owner @{user_cache_dirs}/plasma-svgelements.@{rand6} l -> @{user_cache_dirs}/#@{int},
```
Comment (roddhjav) [inline apparmor.d/groups/kde/kwin_wayland:86]: Simplify this into:
```
<basepath>.lock rwk,
<basepath>{,.@{rand6}} rwl -> <dst>,
```
(Same in other occurrence of this structure)

File: apparmor.d/groups/systemd/systemd-logind [hunk 3]
```diff
@@ -128,6 +128,7 @@ profile systemd-logind @{exec_path} flags=(attach_disconnected,complain) {
   @{sys}/firmware/efi/efivars/OsIndicationsSupported-@{uuid} r,
   @{sys}/fs/cgroup/memory.max r,
   @{sys}/fs/cgroup/memory/memory.limit_in_bytes r,
+  @{sys}/fs/cgroup/system.slice/systemd-logind.service/memory.pressure rw,
```
Comment (roddhjav) [inline apparmor.d/groups/systemd/systemd-logind:131]: Already part of `systemd-common`

File: apparmor.d/groups/kde/kwin_wayland (hunk 1 above)
Comment (curiosityseeker) [reply apparmor.d/groups/kde/kwin_wayland:77]: I had looked into this abstraction. But is `le64` covered by `[a-f0-9]*` ?

**Comment by roddhjav** [comment]
Thanks a lot. Merged!

**Pull request merged**